Reconstruction/reconstruction.tex

```diff
@@ -396,8 +397,8 @@ \subsection{Missing Tranverse Energy}
 In a perfectly reconstructed event, non-zero \met\ implies the presence of neutrinos or DM candidates; however, the failure to properly reconstruct energy deposits or the reconstruction of PF candidates with incorrect energy results in events with large amount of fake \met.
 
 One last cleaning of the PF candidates is conducted in an attempt to fix these events.
-To remove muons from cosmic rays, muon candidates with trajectories more than 1\cm away from the beam axis are removed if the measured \met\ is consequently reduced by half.
-For muons with $\pt > 20\GeV$, the choice of subdetector used to estimate momemtum is reviewed and the smallest available estimate used if it reduces the measured \met\ by half.
+To remove muons from cosmic rays, muon candidates with trajectories more than 1\cm away from the beam axis are removed if the measured \met\ is consequently reduced by half. \PH{I don't think consequently is needed}
+For muons with $\pt > 20\GeV$, the choice of subdetector used to estimate momemtum is reviewed and the smallest available estimate used if it reduces the measured \met\ by half. \PH{rephrase: the if it reduces confuseds me}
```

Comment:
guess I was trying not to repeat the previous sentence, but it definitely sounds better that way